package.json

```diff
@@ -152,5 +152,72 @@
     "collectCoverage": true,
     "coverageDirectory": "./coverage/",
     "testEnvironment": "node"
+  },
+  "commitlint": {
+    "rules": {
+      "body-leading-blank": [
+        1,
+        "always"
+      ],
+      "footer-leading-blank": [
+        1,
+        "always"
+      ],
+      "header-max-length": [
+        2,
+        "always",
+        72
+      ],
+      "scope-case": [
+        2,
+        "always",
+        "lower-case"
+      ],
+      "subject-case": [
```

Comment:
What's your reasoning here? I feel like we want it to be always lower case

Reply:
Realizing that this probably looks word by word, it makes sense to allow other cases to include variable names in commit messages